templates/service/systemd/mesos.service.erb

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Mesos <%= @name %>
+After=network.target
+Wants=network.target
+
+[Service]
+Delegate=yes
+ExecStart=/usr/sbin/mesos-<%= @name %> <% if @name=='slave' -%>-containerizers=docker,mesos<% end -%>
```

Comment:
If you're using Mesosphere packages it would be easier to use:
```
ExecStart=/usr/bin/mesos-init-wrapper <%= @name %>
```
if you want to execute directly mesos binaries, we have to make sure, that all environment variables are defined.
Either way `-containerizers=docker,mesos` should not be hardcoded.